esp/templates/program/newprogram_review.html

```diff
@@ -38,13 +38,15 @@ <h2>Program Creation Review</h2>
 <div id="catalog">
 <table align="center">
 <tr>
-<th colspan="2">The following program modules will be added:</th>
+<th colspan="2" class="materials">The following program modules will be added:</th>
 </tr>
-<tr><th class="materials" colspan="2">Module Name</th></tr>
-{% for item in modules %}
+<!-- This loop puts the modules in two columns (so the list isn't too long)
```

Comment:
nit: better to use `{# ... #}` or `{% comment %} ... {% endcomment %}` comments, so it doesn't end up in the rendered HTML at all.